### DISCUSSION TITLE
Couchdb database directory  take  too much space

### DISCUSSION THREAD

**Discussion by guanfeix** [apache/couchdb#3954] [Q&A]
OS: Ubuntu 14.04.1
KernelVersion: 4.19.90
Architecture:x86_64
CouchDB version: 1.6.1
Erlang version: R16B03
It's a single node,a little traffic,maybe about sixty dbs,each with around ten view_functions average
Our Couchdb database directory  is /var/lib/couchdb
After couchdb restart , df -h to check free 30G,even du -sh  show no change in disk take up.
Find the question(root file system space is not enough):
![图片](https://user-images.githubusercontent.com/34960209/157635627-570ca866-c2c6-4d2b-b9ce-241a8511dffd.png)

After restart couchdb:

![图片](https://user-images.githubusercontent.com/34960209/157635543-7a35d095-dc95-4af7-8bef-a3cdfd2a3e6e.png)

**Comment by big-r81** [comment]
Hi,

did you check the space /var/lib/couchdb directly, that you can be sure it's CouchDB? BTW, your are using a really old and unsupported CouchDB version.

**Reply by guanfeix** [reply]
yes,I check it  already.  /var/lib/couchdb  top 20 biggest db file  was the same.This directory basically no change .I test for multiple times. Its real release a lot of memory space which i think it depend on the couchdb process existing times.
we are doing the upgradation job of couchdb.some old version existes for history reason

**Comment by big-r81** [comment]
Maybe you should check if CouchDB (or the erlang vm (beam)) is allocating big files while running, or if other processes doing somethin weird...

**Reply by guanfeix** [reply]
Yes, a lot of views in some of most common use database. And I learn about difference between df and du cmd, is  phantom file.So when use lsof|grep deleted, I found a lot of such  deleted file in ubuntu.But how can I to avoid such use by that.
![图片](https://user-images.githubusercontent.com/34960209/165701763-2fd1438d-1dee-4fb0-a5d9-a79442e41a67.png)
